dist/clouduploader.rb

```diff
@@ -19,6 +19,7 @@
 data_path = ARGV[3]
 filename = ARGV[4]
 data = JSON.parse(File.read(data_path))
+jobid = File.basename(image_path, '.file')
```

Comment:
Maybe we can talk with @mlschroe to just call the file ``result``, then we do not need to construct the filename here. As this should always come from the backend, it shouldn't be a problem but I don't like to construct filenames like we do here ```File.open("#{jobid}.result", "w+")```. Maybe this could introduce some security issue in the future.